docs/getting-started/tutorial-inspect.asciidoc

```diff
@@ -0,0 +1,24 @@
+[[tutorial-inspect]]
+=== Inspecting the data
+
+The visualizations form a picture of the data, but sometimes you need to 
+view the raw data. Perhaps something in the visualization doesn’t seem right 
```

Comment:
I wouldn't call it raw data, that can be a bit confusing. Raw data are the actual individual documents that you see in discover. Instead, this is really the data behind the visualization or the aggregated / bucketed data. I'm not sure how best to phrase this but raw data can be a bit misleading.